pages/common/unzstd.md

```diff
@@ -0,0 +1,16 @@
+# unzstd
+
+> Decompress files with Zstandard compression.
+> More information: <https://github.com/facebook/zstd>.
+
+- Extract zst file(s) (for multiple files, separate file paths by spaces):
+
+`unzstd {{file(s)}}`
+
+- Extract zst file to given path:
+
+`unzstd {{compressed_file}} -o {{path/to/put/extracted_file}}`
```

Comment:
Perhaps this better
```suggestion
`unzstd {{path/to/compressed.ztd}} -o {{path/to/output_directory}}`
```

Reply:
I applied this, but did `extracted_file` instead of `output_directory`.  `unzstd` only allows operating on a single file (as far as I know):

```text
*** zstd command line interface 64-bits v1.5.2, by Yann Collet ***
Usage : 
      unzstd [args] [FILE(s)] [-o file] 

FILE    : a filename 
          with no FILE, or when FILE is - , read standard input
Arguments : 
 -#     : # compression level (1-19, default: 3) 
 -d     : decompression 
 -D DICT: use DICT as Dictionary for compression or decompression 
 -o file: result stored into `file` (only 1 output file) 
 -f     : disable input and output checks. Allows overwriting existing files,
          input from console, output to stdout, operating on links,
          block devices, etc.
--rm    : remove source file(s) after successful de/compression 
 -k     : preserve source file(s) (default) 
 -h/-H  : display help/long help and exit 
 ```

Reply:
Oh, you're right, I forgot that the Arch Linux packages are a tar then compressed (`.pkg.tar.ztd`)